modules/seqexec/engine/src/main/scala/seqexec/engine/Step.scala

```diff
@@ -101,11 +104,10 @@ object Step {
     /**
       * Obtain the resulting `Step` only if all `Execution`s have been completed.
       * This is a special way of *unzipping* a `Zipper`.
-      *
       */
     val uncurrentify: Option[Step[F]] =
       if (pending.isEmpty) focus.uncurrentify.map(
-        x => Step(id, breakpoint, Skipped(false), skipMark, x :: done)
+        x => Step(id, breakpoint, Skipped(false), skipMark, x.prepend(done))
```

Comment:
Isn't this switched around now? Is there any advantage to it?

Reply:
I guess it doesn't matter because you want all the `Execution`s to be done, but I'm just curious.